src/main/scala/codacy/stylelint/Stylelint.scala

```diff
@@ -72,7 +72,7 @@ object Stylelint extends Tool {
     val configuration = configurationFile ++ configurationBaseDirectory
     val formatter = List("--formatter", "json")
 
-    val command = List("stylelint") ++ fileArgument ++ configuration ++ formatter
+    val command = List("npx", "--no-install", "stylelint") ++ fileArgument ++ configuration ++ formatter
```

Comment:
Use the script directly instead of depending on `npx`

Reply:
Nice catch! I was using it to ensure stylelint was in PATH or in local node_modules. But now it is not needed 👍